installers/install.sh

```diff
@@ -128,7 +128,7 @@ else
 fi
 
 # Fetch version info.
-$FETCH $INSTALLERTMPDIR/info.json $JSONURL || exit 1
+$FETCH $INSTALLERTMPDIR/info.json $JSONURL
```

Comment:
We don't appear to use `set -e` for install.sh, so wouldn't this just continue execution if the fetch failed?

Reply:
It would, but the very next line catches the failed download.

Reply:
It looks for the word "Invalid" in `info.json`, which seems like a very specific type of error.

I think we need to:

- Exit if $FETCH had a non-zero exit code
- Ensure the $FETCH error was relayed

It looks like we silence wget and curl, so maybe we can't relay the error without also exposing the user to the verbosity of those commands (worth verifying), but we should probably still at least check for a non-zero exit code.

Reply:
I've added an explicit check for exit status. From my testing, I'm not getting any error output from the $FETCH command, with or without `set -e`.

By the way, the grep test would fail if the file does not exist (which would be the case if the fetch command failed), hence why I didn't feel the need for an explicit exit status check.